ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -154,6 +164,29 @@ class NavigationControllerDemoController: DemoController {
 
         return controller
     }
+    
+    private func changeStyle(in navigationItem: UINavigationItem) {
```

Comment:
can the function name be little bit more descriptive that it will change per second?